source-boilerplate/boilerplate.go

```diff
@@ -21,17 +22,27 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+const LISTEN_TIMEOUT_SECONDS = 30
```

Comment:
Is there a reason the timeout is different for sources vs materializations?

Reply:
Sorry I was going back and forth between testing this PR and another investigation, they are meant to be the same, I just reconsolidated them